compiler/compile/OMRCompilation.hpp

```diff
@@ -969,15 +965,15 @@ class OMR_EXTENSIBLE Compilation
    void reportFailure(const char *reason);
 
 protected:
+   // Signature of the method we are compiling -- keep this at offset 0
```

Comment:
Perhaps incorporating a piece of the PR comment would make sense as to _why_ one would want to keep this field at offset 0, i.e.

> _signature should still be the first member, as this can significantly ease defect investigations.